src/config.c

```diff
@@ -122,6 +122,13 @@ configEnum propagation_error_behavior_enum[] = {{"ignore", PROPAGATION_ERR_BEHAV
                                                 {"panic-on-replicas", PROPAGATION_ERR_BEHAVIOR_PANIC_ON_REPLICAS},
                                                 {NULL, 0}};
 
+configEnum log_format_enum[] = {{"default", LOG_FORMAT_DEFAULT}, {"logfmt", LOG_FORMAT_LOGFMT}, {NULL, 0}};
+
+configEnum log_timestamp_format_enum[] = {{"default", LOG_TIMESTAMP_DEFAULT},
+                                          {"iso8601", LOG_TIMESTAMP_ISO8601},
+                                          {"unix", LOG_TIMESTAMP_UNIX},
```

Comment:
```suggestion
configEnum log_timestamp_format_enum[] = {{"legacy", LOG_TIMESTAMP_DEFAULT},
                                          {"iso8601", LOG_TIMESTAMP_ISO8601},
                                          {"milliseconds", LOG_TIMESTAMP_UNIX},
```
@azuredream Was there a reason we added the unix milliseconds timestamp option? While discussing it, we weren't sure if it was needed.

Reply:
(@madolson The PR currently emits it as seconds, not milliseconds, since the epoch.)

Reply:
If we want the epoch, I think millisecond timestamp makes a lot more sense. A lot of important events happen sub-second.